internal/backends/winit/Cargo.toml

```diff
@@ -59,14 +59,18 @@ rgb = { version = "0.8.27", optional = true }
 # For the Skia renderer
 i-slint-renderer-skia = { version = "=0.3.3", path = "../../renderers/skia", optional = true }
 
+# For the software renderer
+softbuffer = { version = "0.1.1", optional = true }
+raw-window-handle-old = { package = "raw-window-handle", version = "0.4.1", features = ["alloc"] }
```

Comment:
should be optional ?

Reply:
For reasons I don't understand, it seems thatI can remove the dependency altogether again :)